protected/humhub/migrations/m240531_123502_create_profile_banner_thumb.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use humhub\libs\ProfileBannerImage;
+use Imagine\Image\Box;
+use Imagine\Image\ManipulatorInterface;
+use yii\db\Migration;
+use yii\helpers\FileHelper;
+use yii\imagine\Image;
+
+/**
+ * Class m240531_123502_create_profile_banner_thumb
+ */
+class m240531_123502_create_profile_banner_thumb extends Migration
```

Comment:
@marc-farre Shouldn't the variants be created automatically on demand?

Reply:
Variants are created in both places:
- this migration for banners uploaded before HumHub 1.16-beta.3
- https://github.com/humhub/humhub/pull/7040/files#diff-f27b20c5a4b521218e72ea3afd55fe27466c38a0f5babafc5c48329c610daf8dR99-R101 for banners uploaded from this version and later